clients/callback_client.go

```diff
@@ -7,41 +7,39 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/hashicorp/go-retryablehttp"
 	"github.com/livepeer/catalyst-api/config"
 )
 
-const max_retries = 3
-const backoff_millis = 200
-
 type CallbackClient struct {
-	httpClient *http.Client
+	httpClient *retryablehttp.Client
```

Comment:
WDYT of keeping this as a regular `http.Client`, just so we can switch from the retry lib more seamlessly?

Reply:
`http.Client` isn't an interface though, how would we switch between the two?

Reply:
There's apparently a function in the retry client to return a corresponding http.Client! I saw it mentioned in the end of their readme I think

Reply:
Huh nice, I had seen that but had missed that it includes the retry functionality

> StandardClient returns a stdlib *http.Client with a custom Transport, which shims in a *retryablehttp.Client for added retries. 

Will make a new PR for that